### ISSUE TITLE
Missing import in generated mock file

### ISSUE THREAD

**Issue by la55u** [anymaniax/orval#594]
## What are the steps to reproduce this issue?

1. generate the client from the example spec and config below
2. inspect the resulting `api.msw.ts` file

<details>
<summary>openapi.yml</summary>

```yaml
openapi: 3.0.3
info:
  title: My API documentation
  description: My API documentation
  version: "1.0"

tags:
  - name: Notification
    description: Notification controller

components:
  schemas:
    BasePage:
      type: object
      required:
        - totalPages
      properties:
        totalPages:
          type: integer
          format: int32
          default: 0
    NotificationResponse:
      required:
        - id
        - title
      type: object
      properties:
        id:
          type: string
        title:
          type: string
        type: 
           $ref: '#/components/schemas/NotificationType' 
    PageNotificationResponse:
      required:
        - content
      allOf:
        - $ref: "#/components/schemas/BasePage"
      type: object
      x-spring-paginated: true
      properties:
        content:
          type: array
          items:
            $ref: "#/components/schemas/NotificationResponse"
    NotificationType:
      type: string
      enum:
        - TODO


paths:
  /notifications:
    get:
      security:
        - BearerAuth: []
      tags:
        - Notification
      summary: get pageable notifications
      operationId: getNotifications
      x-spring-paginated: true
      responses:
        "200":
          description: Ok
          content:
            application/json:
              schema:
                $ref: "#/components/schemas/PageNotificationResponse"


```
</details>

<details>
<summary>orval.config.ts</summary>

```typescript
export default defineConfig({
  api: {
    input: {
      validation: false,
      target: "./openapi.yml",
    },
    output: {
      workspace: ".",
      target: "generated/api.ts",
      mode: "split",
      client: "react-query",
      headers: false,
      mock: true,
      clean: ['generated'],
      prettier: true,
      override: {
        mock: {
          arrayMin: 5,
          arrayMax: 50,
        },
      },
    },
  },
});
```
</details>

## What happens?

`NotificationType` import is missing, cannot build the code.

<details>
<summary>api.msw.ts</summary>

```typescript
/**
 * Generated by orval v6.10.1 🍺
 * Do not edit manually.
 * My API documentation
 * My API documentation
 * OpenAPI spec version: 1.0
 */
import { rest } from "msw";
import { faker } from "@faker-js/faker";

export const getGetNotificationsMock = () => ({
  totalPages: faker.datatype.number({ min: undefined, max: undefined }),
  content: Array.from({ length: faker.datatype.number({ min: 5, max: 50 }) }, (_, i) => i + 1).map(
    () => ({
      id: faker.random.word(),
      title: faker.random.word(),
      type: faker.helpers.arrayElement([
        faker.helpers.arrayElement(Object.values(NotificationType)),
        undefined,
      ]),
    }),
  ),
});

export const getMyAPIDocumentationMSW = () => [
  rest.get("*/notifications", (_req, res, ctx) => {
    return res(
      ctx.delay(1000),
      ctx.status(200, "Mocked status"),
      ctx.json(getGetNotificationsMock()),
    );
  }),
];

```
</details>

## What were you expecting to happen?

`import { NotificationType } from './api.schemas'`

## Any logs, error output, etc?

No errors.

## Any other comments?
In the schemas there is a `type NotificationType` type and a `const NotificationType` with disabled eslint rule. Might be related?

## What versions are you using?
orval: 6.10.1

**Comment by anymaniax** [comment]
sorry! Fixed will be in the next release

**Comment by anymaniax** [comment]
should be fixed with `6.10.2`

**Comment by la55u** [comment]
thank you, it works!
